models/issues/review.go

```diff
@@ -269,6 +270,12 @@ func FindReviews(ctx context.Context, opts FindReviewOptions) ([]*Review, error)
 	if opts.Page > 0 {
 		sess = db.SetSessionPagination(sess, &opts)
 	}
+	if opts.LatestOnly {
+		sess.In("id", builder.
+			Select("max ( id ) ").
+			From("review").
+			GroupBy("reviewer_id"))
```

Comment:
No more "where" condition for this query?

For a instance with 100,000 reviews, this "IN" query will group these 100,000 reviews for each FindReviews call?

Reply:
issue_id condition is necessary.

Reply:
Done. I wasn't sure about this one since IssueID will not always be provided. I think re-using the outer condition on where on the subquery should be okay?